domain/modeldefaults/service/service.go

```diff
@@ -57,8 +57,9 @@ type State interface {
 	// is returned.
 	UpdateCloudDefaults(ctx context.Context, cloudUID cloud.UUID, attrs map[string]string) error
 
-	// DeleteCloudDefaults deletes the specified cloud default
-	// config values for the provided keys if they exist.
+	// DeleteCloudDefaults will delete the specified default keys from the cloud
```

Comment:
...default config keys....

To be consistent with the comments elsewhere.

Reply:
Done & Done